parachain/src/chain_spec.rs

```diff
@@ -879,7 +879,7 @@ fn interlay_mainnet_genesis(
             replace_btc_dust_value: DEFAULT_DUST_VALUE,
         },
         vault_registry: interlay_runtime::VaultRegistryConfig {
-            minimum_collateral_vault: vec![(Token(KSM), 0)],
+            minimum_collateral_vault: vec![(Token(DOT), 30 * DOT.one())],
```

Comment:
Where was this value decided? This is not the current value on the network. The current value is 0 _KSM_ by the way, we should remove that value

Reply:
cc: @nud3l